packages/editor/src/components/post-title/index.js

```diff
@@ -176,7 +177,10 @@ function PostTitle( _, forwardedRef ) {
 				...create( { html: title } ),
 				...selection,
 			};
-			const newValue = insert( value, create( { html: content } ) );
+			const newValue = insert(
+				value,
+				create( { html: stripHTML( content ) } )
```

Comment:
When I copy/pasted it resulted in a string, so I added this line to catch that case too.

Reply:
@scruffian can you remember what exactly you were talking about here? https://github.com/WordPress/gutenberg/pull/54718 proposes reverting this change but I'm unsure as to whether it's a good idea.

Remember the goal is/was to avoid HTML being inadvertently added to post titles when _pasting_.

Reply:
I think what was happening was if I pasted HTML into the Post Title it could convert the HTML into the equivalent characters, so `<` would become `&lt;`.

Reply:
Thanks 👍